### ISSUE TITLE
Weird error message in F.conv1d

### ISSUE THREAD

**Issue by vadimkantorov** [pytorch/pytorch#22919]
I made some mistake with tensor shape for nn.Conv1d and expectedly got an error. However the error message is quite strange:
```
  File "/miniconda/lib/python3.7/site-packages/torch/nn/modules/conv.py", line 198, in forward
    self.padding, self.dilation, self.groups)
RuntimeError: Expected 3-dimensional input for 3-dimensional weight 768 161 13 140185417077232, but got 4-dimensional input of size [40, 1, 161, 1923] instead
```

Not the giant number `140185417077232`. Looks like some access of uninitialized memory or a problem with printf. Also square brackets and commas are missing in the first weight tensor shape.

Torch version is `1.2.0.dev20190607`

**Comment by SsnL** [comment]
likely a duplicate of https://github.com/pytorch/pytorch/issues/19947

**Comment by vadimkantorov** [comment]
You are right. Then I'll make this issue only about tensor shape formatting.

**Comment by fmassa** [comment]
So @vadimkantorov this issue is about adding `[` in the error message?

**Comment by vadimkantorov** [comment]
@fmassa Yes, the rest is duplicate of @SsnL's issue
